operations/rds-intelligent-scaling-cron/operation.yaml

```diff
@@ -105,4 +112,4 @@ spec:
             namespace: crossplane-system
             name: claude
       retryLimit: 3
-  schedule: '*/2 * * * *'  # Run every 2 minutes for demo
+  schedule: '* * * * *'  # Run every 2 minutes for demo
```

Comment:
Runs every minute though no? 🤔 
And i guess there's no way to infer this from an external source right? So to change the schedule one would need to change the operation itself correct?